pkg/alpine/Dockerfile

```diff
@@ -1,39 +1,37 @@
-ARG ALPINE_VERSION=3.13
-FROM lfedge/eve-alpine:6.2.0 AS cache
+ARG EVE_BUILDER_IMAGE=lfedge/eve-alpine:6.2.0
```

Comment:
This confuses the heck out of me. How can `pkg/alpine/Dockerfile` depend upon `lfedge/eve-alpine:<something>`? Isn't this the Dockerfile used to _create_ `lfedge/eve-alpine:<anything>`?

Reply:
What _is_ `EVE_BUILDER_IMAGE` meant to represent?

Reply:
I get what it is in all other packages; it is the eve base for all other packages. But what meaning does it have in `pkg/alpine`? That _is_ the base.

Reply:
Well, eve-alpine package is VERY special -- so there's a reason why it appears to be confusing. I think this documentation should answer most of your questions https://github.com/lf-edge/eve/blob/master/docs/BUILD.md#building-packages and if not -- the real ask is to improve that doc (please open a separate Issue in EVE describing what's not clear about it)

Reply:
Yeah, I will open a separate issue. I remember that doc - and I just reread it - and I still am confused.

Reply:
Opened and linked